Kubernetes/legos/k8s_get_expiring_certificates/k8s_get_expiring_certificates.py

```diff
@@ -41,32 +55,42 @@ def k8s_get_expiring_certificates(handle, namespace:str='', expiring_threshold:i
     """
     result = []
     all_namespaces = [namespace]
+
     cmd = "kubectl get ns  --no-headers -o custom-columns=':metadata.name'"
-    if namespace is None or len(namespace)==0:
-        kubernetes_namespaces = k8s_kubectl_command(handle=handle,kubectl_command=cmd )
-        replaced_str = kubernetes_namespaces.replace("\n"," ")
-        stripped_str = replaced_str.strip()
-        all_namespaces = stripped_str.split(" ")
+
+    if namespace is None or len(namespace) == 0:
+        response = handle.run_native_cmd(cmd)
+    if response is None:
+        print(
+            f"Error while executing command ({cmd}) (empty response)")
+
+    if response.stderr:
+        raise ApiException(
+            f"Error occurred while executing command {cmd} {response.stderr}")
+    kubernetes_namespaces = response.stdout
+    replaced_str = kubernetes_namespaces.replace("\n", " ")
+    stripped_str = replaced_str.strip()
+    all_namespaces = stripped_str.split(" ")
```

Comment:
this should be part of the if condition above? ie

`if namespace is None or len(namespace) == 0:`

Reply:
That is correct, changed the alignment.